README.md

```diff
@@ -7,12 +7,15 @@
 
 Paynow PHP Library provides access to Paynow API from Applications written in PHP language. 
 
+## Requirements
+PHP 7.1 or higher
+
 ## Installation
 
 ### Composer
 Install the library using [Composer](https://getcomposer.org)
 ```bash
-$ composer require pay-now/paynow-php-sdk
+$ composer require pay-now/paynow-php-sdk nyholm/psr7 php-http/curl-client
```

Comment:
also mention that other libs can be used not just nyholm

Reply:
@amodliszewski ready